src/app/contextBroker/contextBroker.cpp

```diff
@@ -325,6 +328,13 @@ PaArgument paArgs[] =
 
 
 //
+// /v2 conv-ops 
```

Comment:
Taking into account @jmcanterafonseca comment, maybe it is better just `/v2 operations`.